### PR TITLE
Honor removed review request

### LINKED ISSUE #9: Unable to dismiss reviewer
For some directories, there's a bunch of different stuff in there and I'm not competent to review all of it, so I sometimes want to dismiss myself as a reviewer. However, when there's another push to the branch wpt-pr-bot readds me as a reviewer because I appear in a OWNERS file.

### PR DESCRIPTION
**Pull request by zcorpan**
Fixes #9.

### CODE REVIEW

File: lib/metadata/find-removed-reviewers.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"use strict";
+var github = require('../github');
+
+function findRemovedReviewers(number) {
+    return github.get("/repos/:owner/:repo/issues/:number/events", {number: number})
+        .then(function(response) {
+            if (!response /* TODO or something */) {
```
Comment (jugglinmike) [inline lib/metadata/find-removed-reviewers.js]: This isn't very actionable. Can you be more specific about what you'd like to see done?

File: lib/metadata/find-removed-reviewers.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                throw new Error("Something's up!")
```
Comment (jugglinmike) [inline lib/metadata/find-removed-reviewers.js]: How about "Empty response received from GitHub API" ?

File: lib/metadata/find-removed-reviewers.js (hunk 1 above)
Comment (zcorpan) [reply lib/metadata/find-removed-reviewers.js]: I wasn't sure what shape response would have when there is an error, and haven't tested for that.

**Comment by zcorpan** [comment]
OK addressed, also I had missed to add the test, oops.

File: test/find-removed-reviewers.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+var assert = require('assert'),
+    findRemovedReviewers = require('../lib/metadata/find-removed-reviewers');
+
+suite('integration', function() {
+    var number = "11512";
+    test('directory contains file', function() {
+        return findRemovedReviewers(number)
+          .then(function(reviewers) {
+              reviewers.sort();
```
Comment (jugglinmike) [inline test/find-removed-reviewers.js]: This can be removed

Comment (zcorpan) [reply test/find-removed-reviewers.js]: done

**Comment by jugglinmike** [comment]
Good going, @zcorpan!

**Pull request merged**